presto-docs/src/main/sphinx/functions/json.rst

```diff
@@ -71,6 +71,32 @@ Cast from JSON
         SELECT CAST(JSON '{"k1": [1, 23], "k2": 456}' AS MAP(VARCHAR, JSON)); -- {k1 = JSON '[1,23]', k2 = JSON '456'}
         SELECT CAST(JSON '[null]' AS ARRAY(JSON)); -- [JSON 'null']
 
+.. note::
+
+    When casting from ``JSON`` to ``ROW``, by default the case of double quoted field names
+    in RowType is strictly enforced when matching, and unquoted field name in RowType is
+    case-insensitive when matching. For example::
+
+        SELECT CAST(JSON '{"v1":123,"V2":"abc","v3":true}' AS ROW(v1 BIGINT, v2 VARCHAR, v3 BOOLEAN)); -- {v1=123, v2=abc, v3=true}
+        SELECT CAST(JSON '{"v1":123,"V2":"abc","v3":true}' AS ROW(v1 BIGINT, "V2" VARCHAR, v3 BOOLEAN)); -- {v1=123, V2=abc, v3=true}
+        SELECT CAST(JSON '{"v1":123,"V2":"abc","v3":true}' AS ROW(v1 BIGINT, "v2" VARCHAR, v3 BOOLEAN)); -- {v1=123, v2=null, v3=true}
+        SELECT CAST(JSON '{"v1":123,"V2":"abc", "v2":"abc2","v3":true}' AS ROW(v1 BIGINT, v2 VARCHAR, "V2" VARCHAR, v3 BOOLEAN)); -- {v1=123, v2=abc2, V2=abc, v3=true}
+
+    If the name of a field does not match (including case sensitivity), the value is null.
+
+    To totally ignore the case of field names in RowType when casting from ``JSON`` to ``ROW``
+    (for example, for legacy support) set config property ``legacy_json_cast`` to ``true``
+    in the coordinator and the worker's `config properties <https://prestodb.io/docs/0.285/installation/deployment.html#config-properties>`_.
```

Comment:
```suggestion
    in the coordinator and the worker's `config properties <../installation/deployment.html#config-properties>`_.
```

Fixed link from an absolute link to an older version of the docs to a link. Verified suggestion in a local build. 

As a suggestion: the destination of the link ([Config Properties](https://prestodb.io/docs/current/installation/deployment.html#config-properties) in Deploying Presto) does not mention the config property ``legacy_json_cast``. Suggest you consider adding the config property ``legacy_json_cast`` to [Properties Reference](http://prestodb.io/docs/current/admin/properties.html), and changing this link to point to the section of [Properties Reference](http://prestodb.io/docs/current/admin/properties.html) that you add it to.

Reply:
Hi @steveburnett, thanks for your suggestion, I'm not so sure whether it's appropriate to add `legacy_json_cast` to [Properties Reference](http://prestodb.io/docs/current/admin/properties.html), for the section is used to describe the most important config properties. Feels `legacy_json_cast` is not so important as properties like `join-distribution-type` or `experimental.spill-enabled` etc. What your opinion, do you still think it's appropriate to be added there?

Reply:
@hantangwangd that's a good question! 

To back up a level of abstraction, I feel that the config property `legacy_json_cast` should be documented _somewhere_ in the Presto docs. If you or others disagree with that, I'm open to understanding why, and if we conclude that we don't need to document, it then we don't need to worry about where to not document it :). 

Assuming that `legacy_json_cast` should be documented in the Presto docs, let's discuss where. 

I feel that [Properties Reference](http://prestodb.io/docs/current/admin/properties.html) should be more complete of a reference than it currently is, with the primary exclusion factor being the stated "not include any connector-specific catalog configuration properties". 

For example, I was planning to update [Properties Reference](http://prestodb.io/docs/current/admin/properties.html) with the properties in [Config Properties](https://prestodb.io/docs/current/installation/deployment.html#config-properties) in Deploying Presto at some point. I've thought about investigating if we could automate generating a list of config properties from the current build of Presto, and automatically publishing that list on this page in a "docs-as-code" strategy. Therefore I thought Properties Reference would be a good place to document this new and not-connector-specific config property. 

Do you have a better idea of where to document `legacy_json_cast`? I'm less emphatic about where in the doc than I am that it should be in the documentation. Let me know what you think!

Reply:
@steveburnett Thanks for your detailed description. Agree that we should document the config property `legacy_json_cast`. And since you are planning to update and enlarge [Properties Reference](http://prestodb.io/docs/current/admin/properties.html), then I think it would be appropriate to add `legacy_json_cast` there. Will do it!